docs/usage/faq.md

```diff
@@ -41,21 +41,21 @@ If you're self hosting Renovate, use the latest release if possible.
 
 ## Renovate core features not supported on all platforms
 
-| Feature              | Platforms which lack feature                      | See Renovate issue(s) |
-| -------------------- | ------------------------------------------------- | --------------------- |
-| Dependency Dashboard | BitBucket, BitBucket Server, Azure                | #9592                 |
-| Hosted app           | GitLab, BitBucket, BitBucket Server, Azure, Gitea |                       |
+| Feature              | Platforms which lack feature                      | See Renovate issue(s)                                        |
+| -------------------- | ------------------------------------------------- | ------------------------------------------------------------ |
+| Dependency Dashboard | BitBucket, BitBucket Server, Azure                | [#9592](https://github.com/renovatebot/renovate/issues/9592) |
+| Hosted app           | GitLab, BitBucket, BitBucket Server, Azure, Gitea |                                                              |
 
 ## Major platform features not suported by Renovate
 
 Some major platform features are not supported at all by Renovate.
 
-| Feature name                            | Platform                               | See Renovate issue(s) |
-| --------------------------------------- | -------------------------------------- | --------------------- |
-| Jira issues                             | BitBucket                              | #3796                 |
-| Merge trains                            | GitLab                                 | #5573                 |
-| Cloning repo over SSH selfhosted        | None                                   | #5406                 |
-| Configurable merge strategy and message | None, BitBucket will get support first | #7184                 |
+| Feature name                            | Platform                               | See Renovate issue(s)                                        |
+| --------------------------------------- | -------------------------------------- | ------------------------------------------------------------ |
+| Jira issues                             | BitBucket                              | [#3796](https://github.com/renovatebot/renovate/issues/3796) |
+| Merge trains                            | GitLab                                 | [#5573](https://github.com/renovatebot/renovate/issues/5573) |
+| Cloning repo over SSH selfhosted        | None                                   | [#5406](https://github.com/renovatebot/renovate/issues/5406) |
+| Configurable merge strategy and message | None, BitBucket will get support first | [#7184](https://github.com/renovatebot/renovate/issues/7184) |
```

Comment:
I've just noticed that we closed issue:

- #7184

Do we need to update the table, and if so, how should we update it?

Reply:
In the PR that closes #7184, we have this comment:

> I guess it's just I didn't know how to GitHub! Thanks for the quick education, appreciate it.
> 
> `#10867`, `#10868`, `#10869`, and `#10870` have been created for the remaining platforms.

Should I add the links to those issues into the table?

Reply:
yes, please update those tables and add the issue links. Thanks

Reply:
I've updated the table row description and added links to all the issues mentioned in the comment.